src/Sarif.FunctionalTests/v2/ConverterTestData/FxCop/Readme.md

```diff
@@ -0,0 +1,15 @@
+Notes on producing test examples.
+
+This output is an FxCop run against a debug build of SARIF-SDK 
+
+0. Enlist in the SARIF-SDK. You will update the test files in this version.
+1. Git clone into a second version of the tree, e.g.:
+   git clone https://github.com/microsoft/sarif-sdk sarif-sdk-fxcop-test
+2. cd sarif-sdk-fxcop-test. git checkout git checkout fc172277895ed80cb3c03d0ce66738cff89aad75
+3. BuildAndTest.cmd --Configuration Debug -NoTest (debug to emit in-source suppressions)
+4. Open SarifSdk.FxCop in the non-test enlistment.
```

Comment:
>FxCop [](start = 17, length = 5)

Does this assume they have a command-line version of FxCop installed? If so, mention it at the top. (Or does VS install the necessary shell association for files of type .FxCop? Is there a dependency on a particular version of VS here?)